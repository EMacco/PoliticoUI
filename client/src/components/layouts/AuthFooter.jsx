import React from 'react';

const AuthFooter = () => {
  return (
    <footer className="user-home-footer">
      Politico Website Designed by Okwara Emmanuel &copy; 2019
    </footer>
  );
};

export default AuthFooter;
