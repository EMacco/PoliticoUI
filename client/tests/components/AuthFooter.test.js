import React from 'react';
import AuthFooter from '@components/layouts/AuthFooter';

const setup = () => {
  const wrapper = shallow(<AuthFooter />);
  return wrapper;
};

describe('<AuthFooter />', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = setup();
  });

  it('should render the AuthFooter component correctly', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
    expect(wrapper.find('AuthFooter')).toBeTruthy();
  });
});
