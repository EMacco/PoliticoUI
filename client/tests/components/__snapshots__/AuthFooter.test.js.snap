// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AuthFooter /> should render the AuthFooter component correctly 1`] = `
<footer
  className="user-home-footer"
>
  Politico Website Designed by Okwara Emmanuel Â© 2019
</footer>
`;
